_episodes/01-quickstart.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Quickstart guide"
+teaching: 1
+exercises: 10
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I activate and test the ESMValTool environment?"
```

Comment:
```suggestion
- "How do I load ESMValTool and check the installation?"
```

Reply:
changed to load and check environment - 9f31e4f.